pkg/exchange/binance/parse.go

```diff
@@ -462,6 +467,60 @@ func parseDepthEvent(val *fastjson.Value) (*DepthEvent, error) {
 	return depth, err
 }
 
+type MarketTradeEvent struct {
+	EventBase
+	Symbol   string           `json:"s"`
+	Quantity fixedpoint.Value `json:"q"`
+	Price    fixedpoint.Value `json:"p"`
+
+	BuyerOrderId  int64 `json:"b"`
+	SellerOrderId int64 `json:"a"`
+
+	OrderTradeTime int64 `json:"T"`
```

Comment:
I guess maybe you can use `types.MillisecondTimestamp` here? not quiet sure

Reply:
int64 is used elsewhere in the parse. Follow the same parsing interface of other events.

Reply:
ok